modules/manage/partials/whole-cluster-restore.adoc

```diff
@@ -0,0 +1,210 @@
+----
+
+```bash
+kubectl apply -f redpanda-cluster.yaml --namespace <namespace>
+```
+--
+Helm::
++
+--
+[tabs]
+====
+--values::
++
+.`cluster-restore.yaml`
+[,yaml]
+----
+storage:
+  tiered:
+    <tiered-storage-settings>
+config:
+  cluster:
+    cloud_storage_attempt_cluster_recovery_on_bootstrap: true
+----
++
+```bash
+helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --create-namespace \
+--values cluster-restore.yaml
+```
+
+--set::
++
+```bash
+helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --create-namespace \
+  --set storage.tiered.<tiered-storage-settings> \
+  --set config.cluster.cloud_storage_attempt_cluster_recovery_on_bootstrap=true
+```
+====
+--
+======
+
+- `storage.tiered`: Make sure to configure the target cluster with the same Tiered Storage settings as the failed source cluster.
+- `config.cluster.cloud_storage_attempt_cluster_recovery_on_bootstrap`: Automate cluster restore in Kubernetes. Setting to `true` is recommended when using an automated method for deployment. When bootstrapping a cluster with a given bucket, make sure that any previous cluster using the bucket is fully destroyed, otherwise Tiered Storage subsystems may interfere with each other.
+endif::[]
+
+ifndef::env-kubernetes[]
+Follow the steps to xref:deploy:deployment-option/self-hosted/manual/index.adoc[deploy a new cluster].
+
+NOTE: Make sure to configure the target cluster with the same Tiered Storage settings as the source cluster.
+endif::[]
+
+ifndef::env-kubernetes[]
+=== Restore to target cluster
+
+You can restore data from a source cluster to a target cluster using the xref:reference:rpk/rpk-cluster/rpk-cluster-storage-restore.adoc[`rpk cluster storage restore`] command.
+
+. Restore data from the source cluster:
++
+[,bash]
+----
+rpk cluster storage restore start -w
+----
++
+The wait flag ()`-w`) tells the command to poll the status of the restore process and then exit when completed.
+
+. Check if a rolling restart is required:
++
+[,bash]
+----
+rpk cluster config status
+----
++
+Example output when a restart is required:
++
+[,bash]
+----
+NODE  CONFIG-VERSION  NEEDS-RESTART  INVALID  UNKNOWN
+1     4               true           []       []
+----
+
+. If a restart is required, perform a xref:manage:cluster-maintenance/rolling-restart.adoc[rolling restart].
+
+endif::[]
+
+ifdef::env-kubernetes[]
+=== Verify that the cluster restore is complete
+
+. Run the following command until it returns `inactive`:
++
+[,bash]
+----
+rpk cluster storage restore status
+----
+
+. Check if a rolling restart is required:
++
+[,bash]
+----
+rpk cluster config status
+----
++
+Example output when a restart is required:
++
+[,bash]
+----
+NODE  CONFIG-VERSION  NEEDS-RESTART  INVALID  UNKNOWN
+1     4               true           []       []
+----
+
+. If a restart is required, perform a xref:manage:kubernetes/k-rolling-restart.adoc[rolling restart].
```

Comment:
Link took me to top of page, not rolling restart topic.

Reply:
We need a rebase off `main`. That doc was merged into `main`.